docs/execution-providers/TensorRTRTX-ExecutionProvider.md

```diff
@@ -26,10 +27,35 @@ Currently TensorRT RTX supports RTX GPUs from Ampere or later architectures. Sup
 {:toc}
 
 ## Install
-Please select the Nvidia TensorRT RTX version of Onnx Runtime: https://onnxruntime.ai/docs/install. (TODO!)
+
+### Windows / C/C++
+
+TODO: All sources of download like Nuget, etc. (WinML?)
+
+### Python
+
+```sh
+pip install onnxruntime-gpu
+```
 
 ## Build from source
-See [Build instructions](../build/eps.md#tensorrt-rtx).
+
+### Pre-requisites
+* Download latest [CUDA toolkit](https://developer.nvidia.com/cuda-toolkit)
+  * The path to the CUDA installation must be set via the `CUDA_HOME` environment variable, or use the `--cuda_home` arg in the build command. The installation directory should contain `bin`, `include` and `lib` sub-directories.
+* Download [TensorRT RTX](https://developer.nvidia.com/tensorrt-rtx)
+
+### Windows
+
+```ps
+.\build.bat --config Release --parallel 32 --build_dir build --use_nv_tensorrt_rtx --tensorrt_rtx_home path\to\tensorrt-rtx" --cuda_home "path\to\cuda\home" --cmake_generator "Visual Studio 17 2022" --use_vcpkg --build_shared_lib --enable_cuda_minimal_build --skip_tests --build
+```
```

Comment:
I think we should be able to have a single command line for Windows and Linux. Besides that: 
- `--parallel` is better than `--parallel 32`
- The linux command line also mentions visual studio as generator 
- Are these the actual command lines for a windows ML supported build ? We might want to include DML EP for windows ? 
- Is `enable_cuda_minimal_build` still required ? It should not have impact if CUDA EP is disabled.